src/toil/serviceManager.py

```diff
@@ -159,7 +162,7 @@ def isActive(self, service):
         """
         return self.jobStore.fileExists(service.terminateJobStoreID)
```

Comment:
The error `Returning Any from function declared to return "bool"` mypy throws here should be resolved once the type hints for abstractJobStore.py are merged.